src/mudlet-lua/tests/README.md

```diff
@@ -9,13 +9,33 @@ On **macOS**:
 
 	brew install luarocks
 	luarocks install busted
+	
+On **Windows**
+
+1. Install LuaRocks (Windows all-in-one executable (32-bit) is the version you need)
+	- Unzip the installer
+  - Open the x86 Native Tools Command Prompt that comes with Visual Studio in administrator mode (regular command prompt may work but is untested; the x64 Native Tools Command Prompt will *not* work)
+	- Navigate to the folder containing the unzipped files
+	- `install /P <install_path> /SELFCONTAINED /L` (omit angular brackets)
+	  - You may need to include the `/F` option if you have previously installed LuaRocks
+	- Write down the recommend `LUA_PATH` and `LUA_CPATH` somewhere. You will need them in the running tests section
+	  - Do *not* set `LUA_PATH` or `LUA_CPATH` system environment variables as suggested by the installer output. It will interfere with Mudlet's package loader
+	  - Setting `Path` and `PATHEXT` is fine
+2. Install Busted
+	- Open a command prompt and enter `luarocks install busted`
+	- If you get a command not recognized message
```

Comment:
```suggestion
	- If you get a `command not recognized` message:
```

Reply:
I have changed the text to more accurately reflect the message that would appear.